framework/test/mocha/functional/http/get/blocks.js

```diff
@@ -87,7 +88,7 @@ describe('GET /blocks', () => {
 			});
 
 			it('using invalid height = -1 should fail with error', async () => {
-				return blocksEndpoint.makeRequest({ height: 0 }, 400).then(res => {
+				return blocksEndpoint.makeRequest({ height: -1 }, 400).then(res => {
```

Comment:
Why this change in the height?

Reply:
As we are testing -1 and the value was 0.
Two same tests above each other, probably typo?

Reply:
@pablitovicente Reference: https://github.com/LiskHQ/lisk/blob/development/framework/test/mocha/functional/http/get/blocks.js#L83

Reply:
Ohh then maybe add for zero and correct expectation (maybe split in new issue this part?)

Reply:
Resolved!